config_test.go

```diff
@@ -112,15 +115,19 @@ var _ = Describe("JWTAuth Config", func() {
 				}`, false, []Rule{
 					Rule{
 						Path:        "/",
-						KeyFile:     []string{"/test/test.pub", "/test/test2.pub"},
-						KeyFileType: RSA,
+						KeyBackends: []KeyBackend{&LazyPublicKeyBackend{filename: "/test/test.pub"}, &LazyPublicKeyBackend{filename: "/test/test2.pub"}},
 					},
 				}},
 				{`jwt {
 					path /
-					publickey /test/test.pem
+					publickey /test/test.pub
 					secret /test/test.secret
-				}`, true, nil},
+				}`, false, []Rule{
```

Comment:
This test removes the error you get when you try to mix HMAC and public key methods.  This is mainly to prevent people from accidentally placing a public key in JWT_SECRET which will allow an attacker to trivially forge a valid token using the public key.